src/metabase/driver/ddl/concurrent.clj

```diff
@@ -0,0 +1,18 @@
+(ns metabase.driver.ddl.concurrent
+  (:import [java.util.concurrent Callable Executors ExecutorService Future ThreadFactory]))
+
+(defonce ^:private thread-factory
+  (reify ThreadFactory
+    (newThread [_ r]
+      (doto (Thread. r)
+        (.setName "ddl worker")
+        (.setDaemon true)))))
```

Comment:
We're using `org.apache.commons.lang3.concurrent.BasicThreadFactory$Builder` elsewhere which gives you the benefit of giving different threads different names instead of naming all three threads `ddl worker`

Reply:
I think this threadpool can actually die now that we have moved everything over to quartz. Will verify. We should also carry this note over to the threadpool for sync.

Reply:
Cool it's probably better to let Quartz handle it since it takes care of multiple instances as well as restarting jobs if the instance suddenly goes down or whatever